nextcord/message.py

```diff
@@ -620,6 +621,49 @@ def created_at(self) -> datetime.datetime:
         return utils.snowflake_time(self.id)
 
 
+class MessageCall:
+    """Represents a message's call data.
+
+    .. versionadded:: 3.0
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two message calls are equal.
+
+        .. describe:: x != y
+
+            Checks if two message calls are not equal.
```

Comment:
these haven't been implemented. are they even needed? (currently check identity which might be okay?)